File: baekjoon/문자열.js
const fs = require("fs");
const [a, b] = fs.readFileSync("/dev/stdin").toString().trim().split(" ");

const diff = b.length - a.length;

let min = Infinity;

for (let i = 0; i <= diff; i++) {
  let temp = 0;
  for (let j = 0; j < a.length; j++) {
    if (a.charAt(j) !== b.charAt(j + i)) {
      temp++;
    }
  }
  min = Math.min(min, temp);
}

console.log(min);

File: baekjoon/문자열.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
)

var (
	in  = bufio.NewReader(os.Stdin)
	out = bufio.NewWriter(os.Stdout)
)

func main() {

	var a, b string
	_, _ = fmt.Fscan(in, &a, &b)
	min := int(math.Inf(1))
	diff := len(b) - len(a)
	for i := 0; i <= diff; i++ {
		count := 0
		for j := 0; j < len(a); j++ {
			if a[j] != b[i+j] {
				count++
			}
		}
		if min > count {
			min = count
		}
	}

	fmt.Println(min)

}
